### PR TITLE
Fix mini dashboard loading

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

Fix dashboard loading. For some reason, regular dashboards load OK but loading the mini dashboard (after fix from #10771) shows a blank dashboard. Looking at the page sources, the script block content is truncated. The `all_cards` JS property is incomplete and none of the other properties show.

Rather than dump every card definition into the main page content, I changed this so that the cards would be loaded over AJAX first during the display method and then afterwards, the dashboard content would load. This at least fixes the dashboard to the point of displaying it's items but now I am working on fixing an issue where all of the cards in the mini dashboard are displayed over top each other and take the full size of the dashboard when the data in the database seems correct for position and size.

This requires #10771

### CODE REVIEW

**Comment by cconard96** [comment]
OK... apparently the cards are OK on large screens but stacked on small screens. Even on medium screens, only 3 cards are shown.

File: src/Dashboard/Grid.php [hunk 1]
```diff
@@ -424,7 +424,6 @@ public function show(bool $mini = false)
             rand:        '{$rand}',
             embed:       {$embed_str},
             ajax_cards:  {$ajax_cards},
-            all_cards:   {$cards_json},
```
Comment (cedric-anne) [inline src/Dashboard/Grid.php:427]: `$cards_json` is not used now, so it can be removed, and `$cards = $this->getAllDasboardCards();` can be removed too.

**Review by cedric-anne — APPROVED** [review]

**Review by trasher — APPROVED** [review]

**Comment by trasher** [comment]
Could you please rebase to fix conflict?

**Comment by cconard96** [comment]
The issue this PR tried to fix can no longer be recreated so it is not needed anymore.

**Pull request closed without merging**